packages/core/src/tw/command/handlers/state.ts

```diff
@@ -14,6 +16,49 @@ function moveCursorByModelOffset(serviceRegistry: IServiceRegistry, modelOffset:
     serviceRegistry.getService('state').applyTransformation(tn);
 }
 
+export const alignment: ICommandHandler = async (serviceRegistry, textAlign: IParagraphStyle["textAlign"]) => {
```

Comment:
I'd like to reserve this module (`command/handlers/state`) for general state operations.

I think it'd be better to have a more generic command that applies an attribute to all instances of a particular component type in the current cursor selection.